tests-functional/tests/test_one_to_one_messages.py

```diff
@@ -0,0 +1,77 @@
+from time import sleep
+from uuid import uuid4
+import pytest
+from test_cases import OneToOneMessageTestCase
+from constants import DEFAULT_DISPLAY_NAME
+
+@pytest.mark.rpc
+class TestOneToOneMessages(OneToOneMessageTestCase):
+
+    @pytest.fixture(scope="class", autouse=True)
+    def setup_nodes(self, request):
+        await_signals = [
+            "messages.new",
+            "message.delivered",
+        ]
+        request.cls.sender = self.sender = self.initialize_backend(await_signals=await_signals)
+        request.cls.receiver = self.receiver = self.initialize_backend(await_signals=await_signals)
+
+    @pytest.mark.dependency(name="test_one_to_one_message_baseline")
+    def test_one_to_one_message_baseline(self, message_count=1):
+        pk_receiver = self.receiver.get_pubkey(DEFAULT_DISPLAY_NAME)
+
+        self.sender.send_contact_request([{"id": pk_receiver, "message": "contact_request"}])
+
+        sent_messages = []
+        for i in range(message_count):
+            message_text = f"test_message_{i+1}_{uuid4()}"
+            response = self.sender.send_message([{"id": pk_receiver, "message": message_text}])
+            sent_messages.append((message_text, response))
+            sleep(1)
```

Comment:
😮 Sleep for 1 second is not a good thing, right?
Why do we need it, on receiver we're waiting for signals after this anyway.